docs/modules/device-lab-management/pages/restart-deviceconnect-services.adoc

```diff
@@ -0,0 +1,43 @@
+= Restart deviceConnect services
+:navtitle: Restart deviceConnect services
+
+Sometimes, Hybrid and On-prem customers hosting dedicated devices need to restart deviceConnect services on the Mac mini host to apply configuration changes or software updates. This action can be performed from the Kobiton Portal via the Device Management page.
+
+[NOTE]
+When you restart deviceConnect services, all devices linked to the Mac mini will become inaccessible and any active test sessions will be terminated. Make sure to restart the correct Mac mini and inform all affected users beforehand.
+
+== Prerequisites
+- Gather the internal IP address or hostname of the Mac mini host(s)
+- Kobiton administrator with the ADMIN predefined permission
+
+== Steps to restart deviceConnect services
+*Step 1:* Log in to the Kobiton Portal with an administrator account.
+
+*Step 2:* Click your profile picture or icon at the bottom of the left nav and select *Settings*.
```

Comment:
Avoid using directional instruction (left nav) because we might change the location of the nav bar in the future.

Reply:
Here you can reuse the partial doc `open-settings.adoc` under `profile/partials`

Reply:
![95](https://github.com/kobiton/docs/assets/20087517/f05a92db-cae7-4464-8b29-e208995a8fda)

Used the open-settings.adoc. Had to change the open-settings.adoc a bit. Since it started with First, you have to... something...

But my step was the second step. 

Made it more general by changing the sentence to "Click your profile name and picture. Then select *Settings*."

Also with the existing formating of the page with Step 1: Text, Step 2: Text, and so on. Using include actually broke the formating. We can't use include as Step 1: include::, it doesn't work like that. Had to implement in the next line as:

Step 2:

include::....

And then had to change the formating of the entire page to match.